iocage/cli/fetch.py

```diff
@@ -161,3 +151,14 @@ def cli(**kwargs):
         f"Fetching release '{release.name}' from '{release.mirror_url}'"
     )
     release.fetch()
+
+def is_option_enabled(args, name):
+
+    try:
+        value = args[name]
+        if value:
+            return True
+    except:
```

Comment:
Because value might be falsy and then the return would never be reached. The function is supposed to return False if value is None, "", False or does not exist at all.